README.md

```diff
@@ -64,14 +64,14 @@ members would not write Python source code that also manifests this red flag.
 ## Example of a Technical Skill
 
 * Running Python programs with `pyenv` and `pipenv`
-  * N = `pyenv` and `pipenv` not installed or setup on laptop
-  * I = Unable to run programs with `pyenv` and `pipenv`
-  * A = Able to run programs with `pyenv` and `pipenv`
-  * G = Able to run programs with `pyenv` and `pipenv` using additional commands
+  * :disappointed: = `pyenv` and `pipenv` not installed or setup on laptop
+  * :confused: = Unable to run programs with `pyenv` and `pipenv`
+  * :neutral_face: = Able to run programs with `pyenv` and `pipenv`
+  * :smiley: = Able to run programs with `pyenv` and `pipenv` using additional commands
```

Comment:
Can you be more specific about what the "additional commands and flags" actually are? Other students might be confused about what "additional commands" they should run to in order to achieve the level of  :sunglasses:

Reply:
@enpuyou Hi Enpu. This content was already included originally in the repository by the instructor. The only thing we changed was the letter criteria system to the emoji's, whereas the text is still from the course instructor.

As a result, I don't think it is necessary to change this text.